Sources/SwiftRTCore/operators/AlgebraicField.swift

```diff
@@ -38,67 +38,12 @@ import Numerics
 
 @derivative(of: add)
 @inlinable public func _vjpAdd<S, E>(_ lhs: Tensor<S,E>, _ rhs: Tensor<S,E>)
--> (value: Tensor<S,E>, pullback: (Tensor<S,E>) ->(Tensor<S,E>, Tensor<S,E>))
-where S: TensorShape, E: DifferentiableElement
+    -> (value: Tensor<S,E>, pullback: (Tensor<S,E>) -> (Tensor<S,E>, Tensor<S,E>))
+    where S: TensorShape, E: DifferentiableElement
 {
-    print("calling _vjpAdd both")
     return (lhs + rhs, { v in (v, v) })
 }
 
-@derivative(of: add, wrt: lhs)
-@inlinable public func _vjpAdd<S, E>(_ lhs: Tensor<S,E>, _ rhs: Tensor<S,E>)
--> (value: Tensor<S,E>, pullback: (Tensor<S,E>) ->(Tensor<S,E>))
-where S: TensorShape, E: DifferentiableElement
-{
-    print("calling _vjpAdd wrt lhs")
-    return (lhs, {
-        print("calling _vjpAdd pullback wrt lhs")
-        return $0
-    })
-}
-
-
-//------------------------------------------------------------------------------
-@differentiable(where E: DifferentiableElement)
-@inlinable public func add<S,E>(_ lhs: Tensor<S,E>, _ rhs: E) -> Tensor<S,E>
```

Comment:
Note: I would recommend defining logic for `Tensor.+(_:_:)`, `Tensor.-(_:_:)`, etc directly in their body, rather than forwarding to top-level helper functions (`add`, `subtract`, etc).

This has the following benefits:
- Reduces boilerplate of declaring extra functions.
- Reduce need to forward `@differentiable(wrt: lhs)` attributes.
  - If a top-level helper function exists, you need to declare this attribute on both the static method and the top-level helper function. This is easy to forget and may lead to unexpected derivative efficiency.
  - Otherwise, you need to declare it only on the static method: the single entry point and definition.